bootstrap/src/main/scala/org/scalafmt/bootstrap/Bootstrap.scala

```diff
@@ -77,7 +77,7 @@ object ScalafmtBootstrap {
         logger = Some(logger)
       )
     )
-    val resolution = start.process.run(fetch).unsafePerformSync
```

Comment:
I think we can solve this by using an `AtomicReference[Unit]` as a semaphore, you can only run the process with that reference. That should probably avoid the retry logic.